pkg/destroy/openstack/openstack.go

```diff
@@ -1044,7 +1044,7 @@ func deleteContainers(opts *clientconfig.ClientOpts, filter Filter, logger logru
 		logger.Error(err)
 		return false, nil
 	}
-	listOpts := containers.ListOpts{Full: false}
```

Comment:
Sweet. Could you perhaps add a comment to explain what you're doing here?

Reply:
Good point.